packages/react/index.css

```diff
@@ -0,0 +1,10 @@
+body {
+  font-size: 20px;
+  padding: 1rem;
+  font-family: 'Clarity City', 'Avenir Next', 'Helvetica Neue', Arial, sans-serif;
+}
+
+h1,
+h2 {
+  font-weight: 300;
+}
```

Comment:
nit: for the demo styles we could update to use the new core css bundles. If we import he global.min.css that will have all the typography and layout. Then the body could be `<body cds-text="body" cds-layout="p:lg">`  and not need any custom CSS.